config/samples/core_v1alpha1_humiopdfrenderservice.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: core.humio.com/v1alpha1
```

Comment:
This file has a lot of comments that are meaningless. I suggest removing them unless they are adding context.